css-positioning/absolute.css

```diff
@@ -0,0 +1,50 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+  top: 0;
+  right: 0;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  top: 5px;
+  left: 230px;
```

Comment:
This is a big movement. The emoji is against the right side of the `.meme` element, you may want to start from the `right`